styles/components/_typography.scss

```diff
@@ -6,26 +6,28 @@
 %heading--jumbo,
 .heading--jumbo {
```

Comment:
@alxmrtnz If we're no longer using `.heading--jumbo` anywhere in the templates, we can just remove this entire class.